File: core-java-practice/gcr-codebase/programming-elements/AverageOfTwoNumbers.java
package Dayone;
import java.util.*;
public class AverageOfTwoNumbers {
  public static void main(String[] args) {
    Scanner sc =new Scanner (System.in);
    // take the 1st input
    int a=sc.nextInt();
    //take the 2nd input
    int b=sc.nextInt();
    //take the 3rd input
    int c=sc.nextInt();
    int average=(a+b+c)/2;
    System.out.println("The average is :" + average);
  }
}

File: core-csharp-practice/gcr-codebase/programming-elements/AverageOfTwoNumbers.cs
using System;

namespace ProgrammingElements{
    class AverageOfTwoNumbers{
        static void Main(string[] args){
            // take the 1st input
            int a = int.Parse(Console.ReadLine());
            // take the 2nd input
            int b = int.Parse(Console.ReadLine());
            // take the 3rd input
            int c = int.Parse(Console.ReadLine());
            // calculate average
            double average = (a + b + c) / 3.0;
            Console.WriteLine("The average is: " + average);
        }
    }
}
